CHANGES.rst

```diff
@@ -5,13 +5,13 @@
 4.3.2 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Add support for Python 3.7.
```

Comment:
This should go to 4.4.0 for release, right?

Reply:
I don't think so -- no actual code changes were required, so I don't think this is a new feature in the semver-sense.

I've no strong feelings either way (and I've currently no plans of cutting a new release).

Reply:
I believe @tseaver has stated a preference (in github comments) for updating the minor version number when new minor version numbers of Python are added or removed.